test/CMakeLists.txt

```diff
@@ -2,23 +2,24 @@ cmake_minimum_required(VERSION 3.14)
 
 set(gtest_REQUIREDVERSION "1.13.0")
 
-  find_package(GTest ${gtest_REQUIREDVERSION} )
-
-  if(GTest_FOUND)
-    message(STATUS "Found GTest: ${GTest_VERSION} ${gtest_CONFIG}")
-  else()
-    message(STATUS "Fetching GTest ${gtest_REQUIREDVERSION}...")
-    include(FetchContent)
-    set(gtest_WITH_POST_BUILD_UNITTEST OFF)
-    set(gtest_WITH_TESTS OFF)
-    set(INSTALL_GTEST OFF)
-    set(BUILD_GMOCK OFF)
-    FetchContent_Declare(
-      gtest
-      URL https://github.com/google/googletest/archive/refs/tags/v${gtest_REQUIREDVERSION}.zip
-    )
-    FetchContent_GetProperties(gtest)
-  endif()
+find_package(GTest ${gtest_REQUIREDVERSION} )
+find_package(Threads)
+
+if(GTest_FOUND)
+message(STATUS "Found GTest: ${GTest_VERSION} ${gtest_CONFIG}")
```

Comment:
Why remove the indentation here?

Reply:
That was a bit to much being removed. I'll fix that.